components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -293,7 +299,11 @@ export class Panel extends React.Component<Props, State> {
     const { balance, rates, grants } = this.props.rewardsPanelData.walletProperties
     const publisher: RewardsExtension.Publisher | undefined = this.getPublisher()
     const converted = utils.convertBalance(balance.toString(), rates)
-    const notification = this.getNotification()
+    const notification: any = this.getNotification()
+    const notificationType = notification.notification.type
```

Comment:
can we move this logic into separate function as I see that we will be adding more of this in the future

Reply:
@NejcZdovc done

Reply:
I still see it in the render. To which function it was moved?

Reply:
done, function is here: https://github.com/brave/brave-core/pull/1641/files#diff-4067754668fe25a0ce8194d60d8acf74R220